src/encoded/schemas/tissue_sample.json

```diff
@@ -85,8 +85,12 @@
         "core_size": {
             "title": "Core Size",
             "description": "Size of core sample, if applicable (mm)",
-            "type": "number",
-            "minimum": 0
+            "type": "string",
+            "enum": [
+                "1.5",
+                "3.0",
+                "3.0 Donut"
+            ]
```

Comment:
Incompatible change - if any values of this currently exists they are incorrectly marked as number and casting may not work - may require an upgrader

Reply:
I think an upgrader would be required, however, there are currently no values for this field in the portal so as long as the change gets in before any data is submitted (and the templates reflect this change it should be OK).

Reply:
Yes there are currently no values for this field, as core_size just got added very recently